demos/drawer/modal-drawer.html

```diff
@@ -215,7 +215,12 @@ <h1 class="mdc-typography--headline4">Modal Drawer</h1>
         var mainEl = document.querySelector('.demo-main');
         listEl.addEventListener('click', function(event) {
           drawer.open = false;
-          selectedDestinationEl.textContent = event.target.lastChild.textContent;
+          let listItemTarget = event.target;
```

Comment:
Suggestion: if we can set `pointer-events: none` styles to `list-item__text` I believe we wouldn't we wouldn't have to do this.

Same for mdc-menu.